Guides/Chunked.md

```diff
@@ -96,7 +96,8 @@ collection conforms.
 
 ### Complexity
 
-The eager methods are O(_n_), the lazy methods are O(_1_).
+The eager methods are O(_n_) where _n_ is the number of elements in the
+collection. The lazy methods are O(_n_) because the start index is pre-computed.
```

Comment:
There definitely is a difference in complexity between the eager and lazy functions, but I’m finding it difficult to describe succinctly.

The eager functions are O(_n_) where _n_ is the number of elements in the collection.

The lazy functions on the order of O(_n_) since the start index could require iterating to the very end of the collection. Each iteration of the receiving lazy sequence is also on the order of O(_n_), but total _n_ where _n_ is the number of elements in the collection—not _n_<sup>2</sup>.